centinel/client.py

```diff
@@ -123,8 +125,46 @@ def run(self, data_dir=None):
 
             try:
                 logging.info("Running %s test." % (name))
+
+                root = True
+                if os.geteuid() != 0:
+                    logging.info("Centinel is not running as root, "
+                                 "tcpdump will not start.")
+                    root = False
```

Comment:
But if I exit, the entire experiment will not be run. That's the whole point of checking for root.